styles.css

```diff
@@ -1,6 +1,7 @@
 @import url(./components/Buttons/buttons.css);
 @import url(./components/Avatar/avatar.css);
 @import url(./components/Alerts/alerts.css);
+@import url(./components/Cards/cards.css);
```

Comment:
Nice work Arunesh! Just one comment, a good practice is to use all lower cases letters while naming files and folders. Rest looks good to me :)

Reply:
Sure will fix the folder names. Thanks !